UnitTests/Utilities/GeometryModelRenderer/src/PointCloudGeometryModelRenderer.cpp

```diff
@@ -20,8 +20,8 @@
 PointCloudGeometryModelRenderer::PointCloudGeometryModelRenderer(const buw::ReferenceCounted<buw::IRenderSystem>& renderSystem)
 	: GeometryModelRenderer::GeometryModelRenderer(renderSystem)
 {
-	pointCloudGeometryEffect_ = buw::makeReferenceCounted<oip::PointCloudGeometryEffect>(
-		renderSystem_.get(), viewport_, depthStencilMSAA_, worldBuffer_);
+	pointCloudGeometryEffect_ = buw::makeReferenceCounted<oip::PointCloudEffect>(
+		renderSystem_.get(), viewport_, depthStencilMSAA_, worldBuffer_, viewportBuffer);
```

Comment:
I assume your question concerns this line?

Reply:
I see now in the screen shot :D

Reply:
I presume that you'd need to define the `viewportBuffer` to hold some data; I'm just not sure which ...

Reply:
I guess I found a solution: [2ce16c2](https://github.com/tumcms/Open-Infra-Platform/pull/525/commits/2ce16c29c3b9e06a05add21413868d24947148e0)